### PR TITLE
`schedule.txt` File Import Change & UG Changes

### PR DESCRIPTION
**Pull request by alanlowzies**
This PR contains the following changes:
- [`FileManager.java`] When loading data from `schedule.txt`, plan names will be converted to lower case (thanks @a1021492980 for providing the template!)
- [`WerkIt.java`] Update the JavaDocs of `loadRequiredDirectoryAndFiles()` to reflect the additional step of rewriting all resource files' data.
- [UG] Add an infobox to let users know that plan names will be stored in lowercase form.
- [UG] Add an infobox to let users know that searches are case insensitive.

### CODE REVIEW

**Review by emilysim00 — APPROVED** [review]
Looks good!

File: src/main/java/storage/FileManager.java [hunk 1]
```diff
@@ -639,7 +639,7 @@ public void addFilePlanToList(PlanList planList, String[] planFileDataLine)
     public void addFileScheduleToList(DayList dayList, String[] scheduleFileDataLine)
             throws ArrayIndexOutOfBoundsException, InvalidScheduleException, InvalidPlanException {
         String dayNumber = scheduleFileDataLine[0];
-        String planName = scheduleFileDataLine[1];
+        String planName = scheduleFileDataLine[1].toLowerCase().toLowerCase();
```
Comment (a1021492980) [inline src/main/java/storage/FileManager.java:642]: Is there supposed to have 2 `.toLowerCase()`?

Comment (alanlowzies) [reply src/main/java/storage/FileManager.java]: Thanks for spotting it. It has been fixed in Commit 95f1266.

**Review by Musfirahe0556596 — APPROVED** [review]
V good!

**Pull request merged**